examples/README.md

```diff
@@ -0,0 +1,31 @@
+# Examples
+
+General purpose and introductory examples of Beard.
+
+## Getting started
+
+This application works only on Python 2.x. On Linux should be already installed. 
+On Mac OSX you can install it using Homebrew: `brew install python2`
+
+To keep the installation separated from the general system, is better to use virtualenv: 
+```
+pip install virtualenv
+mkdir virtualenv
+cd virtualenv
+source bin/activate
+```
+
+...and install all the required libraries: 
+
+```
+pip install numpy 
+pip install scipy
+```
```

Comment:
Actually `pytho2n setup.py install` should take care of this.

See https://github.com/inspirehep/beard/blob/master/setup.py#L87